scripts/generators/readmes.js

```diff
@@ -21,17 +21,21 @@ const getIssueLabelLink = l =>
   )}%22+is%3Aopen`;
 
 const labels = {
-  "babel-preset-typescript": getIssueLabelLink("area: typescript"),
   "babel-preset-flow": getIssueLabelLink("area: flow"),
+  "babel-preset-node": getIssueLabelLink("area: node"),
   "babel-preset-react": getIssueLabelLink("area: react"),
+  "babel-preset-typescript": getIssueLabelLink("area: typescript"),
```

Comment:
☝️ simple note: `babel-preset-typescript` is moved here to try to sort area labels alphabetically.